Packages/Web/templates/js/transformAnimations.js.flt

```diff
@@ -1,12 +1,15 @@
 <% // Transform animations
 
 // Returns the css transform value at the given time.
+
+var durationInSeconds = timeline.duration * 1000
+
 func transform(at time)
-    var scaleX = timeline.value(property: "transform.scale.x", layer: layer.layer, time: time)
-    var scaleY = timeline.value(property: "transform.scale.y", layer: layer.layer, time: time)
+    var scaleX = timeline.value(property: "transform.scale.x", layer: layer.layer, time: 0)
```

Comment:
add comment here saying, we dont animate scale, use initial value, set it because it gets overwritten